test/performance/odp_bench_packet.c

```diff
@@ -1839,7 +1839,7 @@ int main(int argc, char *argv[])
 	/* Create packet pool */
 	odp_pool_param_init(&params);
 	params.pkt.seg_len = PKT_POOL_SEG_LEN;
-	params.pkt.len     = TEST_MAX_PKT_SIZE;
+	params.pkt.len     = 2 * TEST_MAX_PKT_SIZE;
```

Comment:
Check on line: 1826:`	} else if (capa.pkt.max_len && capa.pkt.max_len < TEST_MAX_PKT_SIZE) {` should be updated to take into account this change. You could also add a comment why `params.pkt.len = 2 * TEST_MAX_PKT_SIZE` is required.

Otherwise looking good, you can add my reviewed-by tag on the same go.

Reply:
Thanks, I will send the V4 as per the comments.